### PR TITLE
Fix: poperly pass on auto_install argument

### PR DESCRIPTION
**Pull request by frankenjoe**
In our tests with `auto_install=True` we forgot to cover `to_yaml()`, which basically relies on `to_yaml_s()`, but unfortunately, we do not pass on the argument at the moment. This fixes the bug and extends the test to cover it.

### CODE REVIEW

File: tests/test_install.py [hunk 1]
```diff
@@ -36,21 +38,26 @@
...
+
+    path = os.path.join(tmpdir, 'tmp.yaml')
+    with open(path, 'w') as fp:
+        fp.write(yaml_s)
+
     # fails because of missing packages
     with pytest.raises(ModuleNotFoundError):
-        audobject.from_yaml_s(
-            yaml_s,
+        audobject.from_yaml(
```
Comment (hagenw) [inline tests/test_install.py:49]: `from_yaml_s` is now automatically covered by `from_yaml()` correct?

Comment (hagenw) [reply tests/test_install.py:49]: I looked at the code and this is not the case, e.g. https://github.com/audeering/audobject/blob/master/audobject/core/api.py#L55-L83

So I guess we need ttest for `from_yaml_s()` as well?

Comment (frankenjoe) [reply tests/test_install.py:69]: Now tests first `from_yaml_s()`, uninstalls package and repeats for `from_yaml()`. Hopefully, we now everything covered :)

Comment (frankenjoe) [reply tests/test_install.py:69]: Had a final close look and actually there are more places where we do not pass on the `auto_install` argument. Will add some more tests :)

Comment (frankenjoe) [reply tests/test_install.py:71]: I created https://github.com/audeering/audonnx/pull/15 and tested it locally with this branch.

Comment (frankenjoe) [reply tests/test_install.py:71]: ... and it worked :)

Comment (frankenjoe) [reply tests/test_install.py:71]: Removed the WIP again.

**Pull request merged**